tests/cpp/operator/runner/core_op_runner_test.cc

```diff
@@ -46,7 +46,15 @@ static const std::vector<std::pair<std::string, std::string>> test_unary_operato
 
 static const std::vector<std::pair<std::string, std::string>> test_binary_operators = {
   { "elemwise_add", "_backward_add" },
-  { "elemwise_mul", "_backward_mul" }
+  // TODO(Deng, Wenqi): In https://github.com/apache/incubator-mxnet/pull/17754,
+  //  we have changed backward op to graph of ops for computing backward of elemwise_mul,
+  //  but the CoreOpExecutor in tests/cpp/include/test_core_op.h actually has issues
+  //  to support this way even it provides CoreOpExecutor::GetBackward for the case.
+  //  e.g: It actually assumes there is one backward for all kinds of op, but elemwise_mul has two.
+  //  It will get wrong dependency for the second backward in CoreOpExecutor::GetBackwardDependency
+  //  due to "return igrad_entries[0].node;" //  and failed to call CoreOpExecutor::bwd_inputs()
+  //  and CoreOpExecutor::bwd_outpuss() due to "CHECK_EQ(backward_.size(), 1U)";.
+//  { "elemwise_mul", "_backward_mul" }
```

Comment:
I don't know much about this, but I don't think removing an existing test would be a good idea.

@apeforest @larroy @sxjscience Would be better people to take the call.

Reply:
For more explanation,  becaue the failure of the test cases are caused by defects of CoreOpExecutor, I just want a pr to achieve only one task.  Maybe we need a other pr or jira to fix the issuee of  CoreOpExecutor.

Reply:
Yes. That makes sense.
However, I feel the mentioned people are in much better position to make the call.